README.md

```diff
@@ -1,4 +1,262 @@
+  impossible). However, the argument supporting the disposition on the RFC
+  needs to have already been clearly articulated, and there should not be a
+  strong consensus *against* that position outside of the working group. Working group
+  members use their best judgment in taking this step, and the FCP itself
+  ensures there is ample time and notification for stakeholders to push back
+  if it is made prematurely.
+  - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+    a *summary comment* trying to lay out the current state of the discussion
+    and major tradeoffs/points of disagreement.
+  - Before actually entering FCP, *all* members of the working group must sign off;
+  this is often the point at which many working group members first review the RFC
+  in full depth.
+- The FCP lasts ten calendar days, so that it is open for at least 5 business
+  days. It is also advertised widely,
+  e.g. in the [Weekly IPFS All Hands Call](https://github.com/ipfs/pm/#weekly-all-hands). This way all
+  stakeholders have a chance to lodge any final objections before a decision
+  is reached.
+- In most cases, the FCP period is quiet, and the RFC is either merged or
+  closed. However, sometimes substantial new arguments or ideas are raised,
+  the FCP is canceled, and the RFC goes back into development mode.
+
+### Status: Accepted    
+
+Once an RFC becomes "Accepted" then authors may implement it and submit the
+feature as a pull request to the corresponding repo. Being "Accepted" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "Accepted"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether an IPFS developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+### Modifications to Accepted RFCs
+
+Modifications to "Accepted" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the working group to decide.
```

Comment:
I think it's safe to say that everything but cosmetic changes (spell fixes and so on, "changing shape does not change meaning") should be a new RFC, and not up to the working groups to modify RFCs as they see fit

Reply:
Are you suggesting a modification to the text here? I think the current wording allows for the usage that you're describing. We should avoid making this initial document too specific or rigid.